javascript/implementation/IndexedDbStore.ts

```diff
@@ -591,7 +591,9 @@ export class IndexedDbStore implements Store {
      * @param asOf show results as of a time in the past
      * @returns a promise of a list of ChangePairs
      */
-    async getOrderedEntries(container: Muid, through = Infinity, asOf?: AsOf): Promise<Entry[]> {
+    //async getOrderedEntries(container: Muid, through = Infinity, asOf?: AsOf): Promise<Entry[]> {
```

Comment:
Still want to have this comment in on merge?

Reply:
good catch